orttraining/tools/amdgpu/Dockerfile.rocm4.3.1.pytorch

```diff
@@ -0,0 +1,172 @@
+# docker build --network=host --file Dockerfile.rocm4.3.1.pytorch --tag ort:rocm4.3.1-pytorch .
```

Comment:
These files can be much simpler now I would say.
From source example:
https://github.com/microsoft/huggingface-transformers/blob/sukha/mi100-v100-benchmarking/docker/Dockerfile.ort181-deepspeed-torch181-rocm42-dev

From wheel example:
https://github.com/pytorch/ort/blob/main/docker/Dockerfile.ort-torch190-onnxruntime-nightly-rocm4.3.1-ubuntu18.04

Reply:
Thanks, yeah, I noticed these dockerfile. Will try them when working with ROCm 4.5.x